<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.test.automation.uiAutomation.homepage.TC001_verifylogin_withinvalidcredentials" tests="4" time="0.006" ignored="0" failures="0" timestamp="12 Jul 2017 07:09:48 GMT" skipped="3" hostname="Vignesh-PC" errors="1">
  <testcase name="logintest_invalid" time="0.003" classname="com.test.automation.uiAutomation.homepage.TC001_verifylogin_withinvalidcredentials">
    <skipped/>
  </testcase> <!-- logintest_invalid -->
  <testcase name="logintest_invalid" time="0.002" classname="com.test.automation.uiAutomation.homepage.TC001_verifylogin_withinvalidcredentials">
    <skipped/>
  </testcase> <!-- logintest_invalid -->
  <testcase name="endtest" time="0.000" classname="com.test.automation.uiAutomation.homepage.TC001_verifylogin_withinvalidcredentials">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.test.automation.uiAutomation.testbase.Testbase.closebrowser(Testbase.java:179)
	at com.test.automation.uiAutomation.testbase.Testbase.endtest(Testbase.java:169)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
	at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:115)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </error>
  </testcase> <!-- endtest -->
  <testcase name="logintest_invalid" time="0.001" classname="com.test.automation.uiAutomation.homepage.TC001_verifylogin_withinvalidcredentials">
    <skipped/>
  </testcase> <!-- logintest_invalid -->
</testsuite> <!-- com.test.automation.uiAutomation.homepage.TC001_verifylogin_withinvalidcredentials -->
